config.cr

```diff
@@ -21,14 +21,10 @@ Lucky::StaticFileHandler.configure do |settings|
   settings.hide_from_logs = true
 end
 
-Lucky::SessionCookie.configure do |settings|
+Lucky::Session.configure do |settings|
```

Comment:
I assume you are sticking with `Session` instead of `CookieSession`? Or maybe change to that at a later date?

Reply:
Oh I forgot to mention that on the other PR. I was going to rename it all, but then I realized the `Session` object was mostly just a hash like object. There is the static method for creating one from a cookie though. So my thought was that if we had multiple types of session stores, we'd still have the `Session` object but maybe the method would be something like: `Session.from(session_processor, cookies)`. That way the session is pretty much the same, but you can pass in whatever processor you want. Like `Session.from(CookieSessionProcessor, cookies` or `Session.from(RedisSessionProcessor, cookies)`. What do you think?
```

Reply:
Ah I see. Yeah that makes sense. I think I mostly liked SessionCookie or whatever from a teaching aspect: "remember, this is a cookie you're dealing with here, not something else". But it's not a cookie at this point, so that doesn't make sense. 👍